ntoskrnl/ke/i386/traphdlr.c

```diff
@@ -930,6 +980,15 @@ KiTrap0CHandler(IN PKTRAP_FRAME TrapFrame)
     KiSystemFatalException(EXCEPTION_STACK_FAULT, TrapFrame);
 }
 
+DECLSPEC_NORETURN
+VOID
+FASTCALL
+KiTrap0EHandler(IN PKTRAP_FRAME TrapFrame);
+
+_TRAPALLOC_(".inthdlr$0D_A") char _KiTrap0D_start;
+_TRAPALLOC_(".inthdlr$0D_Z") char _KiTrap0D_end;
```

Comment:
These two could be '0D_B' and 'OD_D' even.

Reply:
no.